### PR TITLE
Added sortmerna 3

### PR DESCRIPTION
**Pull request by eggzilla**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/sortmerna/3.0.0/build.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+mkdir -pv $PREFIX/bin
+bash build.sh
```
Comment (bgruening) [inline recipes/sortmerna/3.0.0/build.sh]: It seems there is no build.sh file in the source dir?

File: recipes/sortmerna/3.0.0/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - zlib
+  run:
+    - zlib
```
Comment (bgruening) [inline recipes/sortmerna/3.0.0/meta.yaml]: zlib is exporting themself now, this is a new feature of the zlib package, so ideally you do not need to specify it anymore in the run dependency

File: recipes/sortmerna/3.0.0/build.sh (hunk 1 above)
Comment (eggzilla) [reply recipes/sortmerna/3.0.0/build.sh]: yeah, building now depends on cmake, i will make the necessary modifications :-)

File: recipes/sortmerna/3.0.0/meta.yaml (hunk 2 above)
Comment (eggzilla) [reply recipes/sortmerna/3.0.0/meta.yaml]: removed, thanks for reviewing :-)

**Comment by daler** [comment]
Closing due to inactivity, please re-open if needed.

**Pull request closed without merging**